packages/web-components/fast-components/src/accordion/accordion.ts

```diff
@@ -0,0 +1,129 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowUp,
+    keyCodeEnd,
+    keyCodeHome,
+    wrapInBounds,
+} from "@microsoft/fast-web-utilities";
+import { StartEnd } from "../patterns/start-end";
+import { applyMixins } from "../utilities";
+import { FASTAccordionItem } from "./accordion-item";
+
+export enum AccordionExpandMode {
+    single = "single",
+    multi = "multi",
+}
```

Comment:
Do we know what these are called in other design systems?

Reply:
Some don't even offer the option, One system call the control `collapse` which allows multiple by default and if you set `accordion` too `true it only allows one. A few refer to it as multiple vs single selection

Reply:
FWIW aria uses "multi" in the `aria-multiselectable` attribute